azure-pipelines-build-release.yml

```diff
@@ -0,0 +1,47 @@
+# Continuous integration (CI) triggers cause a pipeline to run whenever you push 
+# an update to the specified branches or you push specified tags.
+    
+trigger:
+  tags:
+    include:
+    - release-*    
+  paths:
+    exclude:
+    - README.md
+
+# Pull request (PR) triggers cause a pipeline to run whenever a pull request is 
+# opened with one of the specified target branches, or when updates are made to 
+# such a pull request.
+#
+# GitHub creates a new ref when a pull request is created. The ref points to a 
+# merge commit, which is the merged code between the source and target branches 
+# of the pull request.
+#
+# Opt out of pull request validation 
+pr: none
+
+# By default, use self-hosted agents
+pool: Default
+
+# Image tag name for Fuse projects
+#parameters:
+#- name: imagetag
+#  displayName: Image tag to be built and/or deployed
+#  type: string
+#  default: latest
+
+resources:
+  repositories:
+  # Azure DevOps repository
+  - repository: tiedonohjaus-pipelines
+    type: git
+    # Azure DevOps project/repository
+    name: tiedonohjaus/tiedonohjaus-pipelines
+
+extends:
+  # Filename in Azure DevOps Repository (note possible -ui or -api)
+  # Django example: azure-pipelines-tiedonohjaus-api-release.yml
+  # Drupal example: azure-pipelines-drupal-release.yml
+  template: azure-pipelines-build-tiedonohjaus-api-release-staging.yml@tiedonohjaus-pipelines
+#  parameters:
+    #imagetag: ${{ parameters.imagetag }}
```

Comment:
Well ok, minor nitpick from unix/linux perspective would be good to have newlines at the end of the yaml files.